engine.py

```diff
@@ -251,6 +251,70 @@ def pre_app_init_nuke(self):
         os.environ["TANK_ENGINE"] = self.instance_name
         os.environ["TANK_CONTEXT"] = sgtk.context.serialize(self.context)
 
+        """
+        https://jira.autodesk.com/browse/SG-25374
+        Weblogin does not show up in Nuke 11 and makes Nuke 12 and 13 to crash
+
+        To avoid Nuke crash, a monkeypatch of on_dialog_closed is required,
+        here the user is warned about restarted nuke is needed to continue.
+        """
+        sgtk.authentication.sso_saml2.core.sso_saml2_core.SsoSaml2Core.on_dialog_closed = (
```

Comment:
This works ?  I was expecting you to need to use `setattr()`

If it works, great !

Reply:
Yes it works well.